suse/macros

```diff
@@ -230,6 +230,11 @@
 %cflags_profile_generate -fprofile-update=atomic -fprofile-generate
 %cflags_profile_feedback -fprofile-use
 
+# Reproducible builds
+%build_mtime_policy clamp_to_buildtime
+%_buildtime %{getenv:SOURCE_DATE_EPOCH_MTIME}
```

Comment:
I need to double check, if this will fail if the env does not exist or is not a number. I think the main failure case when we failed on not having a SDE, was packages without any changelog. See the `%if` in the Factory prconf snippet pasted above.

As we can use Lua here to expand, we can probably do something more clever. If the env var is not sane then we fail if it is for a production distribution or use the highest mtime with warning if it is not.

Might be worth to consider if we can and should move all or part of the related shell code into this as Lua https://github.com/openSUSE/post-build-checks/blob/73dbdbc/suse-buildsystem.sh . Not sure if we have access to the relevant files here.